syft/workers/websocket_client.py

```diff
@@ -10,11 +10,10 @@
 import time
 
 import syft as sy
-from syft.codes import MSGTYPE
 from syft import messaging
-from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+from syft.codes import MSGTYPE
```

Comment:
Should this be replaced with the new message object?

Reply:
see #2531